cli_tools/common/utils/param/param_utils.go

```diff
@@ -173,3 +173,29 @@ func GetGlobalResourcePath(resourceType string, resourceName string) string {
 func GetRegionalResourcePath(region string, resourceType string, resourceName string) string {
 	return getResourcePath(fmt.Sprintf("regions/%v", region), resourceType, resourceName)
 }
+
+// GetZonalResourcePath gets zonal resource path based on either a local resource name or a path
+func GetZonalResourcePath(zone string, resourceType string, resourceName string) string {
+	return getResourcePath(fmt.Sprintf("zones/%v", zone), resourceType, resourceName)
+}
+
+// ReverseMap reverses keys and their values.
+func ReverseMap(m map[string]string) (map[string]string, bool) {
```

Comment:
Since this is in a common util package, I'd want to see an idiomatic API. If you want to keep the boolean optimization, then move it to your package and make it private.